Error Log Update

wiliams727 opened the pull request:
Errors now store:
- Error number
- Email address
- First & Last name (when available)

Type check and Max size length added to some attributes

wiliams727 committed:
Error Log Update

Errors now store:
- Error number
- Email address
- First & Last name (when available)

Type check and Max size length added to some attributes

mishavee commented on growsocial/common/methods/errorLogsMethods.js:
```diff
@@ -1,10 +1,77 @@
+	  	};
+
+	  	// Tag (Type of error) (Max 30 characters)
+	  	if (typeof error.tag === 'string') {
```
I'd rather use the check function, the package is already added. See meteor docs: http://docs.meteor.com/#/full/check_package

mishavee commented on growsocial/common/methods/errorLogsMethods.js:
```diff
@@ -1,10 +1,77 @@
+
+	  	// Message (Max 40 characters)
+	  	if (typeof error.message === 'string') {
+	  		if (error.message.length > 40) {
```
Considering we will log errors from other parts of the app, we will need to be more generous here, say 200 characters.

mishavee commented on growsocial/client/views/login/login.js:
```diff
@@ -124,6 +130,10 @@ Template.register.events({
           message: err.message,
+          errNumber: err.error,
+          email: user.email,
+          fName: user.profile.firstname,
```
Please change fName to firstName, and lName to lastName

wiliams727 committed:
Fix

wiliams727 committed:
Error

“Exception while invoking method ‘addErrorLog’ ReferenceError: Match is
not defined"

wiliams727 committed:
Fix

wiliams727 committed:
Fix
